dev/sparktestsupport/modules.py

```diff
@@ -612,6 +612,13 @@ def __hash__(self):
         "pyspark.pandas.typedef.typehints",
         # unittests
         "pyspark.pandas.tests.test_dataframe",
+        "pyspark.pandas.tests.test_config",
```

Comment:
May I sort these tests in my last PR? That would make resolving other PRs' conflicts much easier.